source/_integrations/aten_pe.markdown

```diff
@@ -0,0 +1,59 @@
+---
+title: "ATEN Rack PDU"
+description: "Instructions on how to integrate ATEN Rack PDUs into Home Assistant."
+ha_category:
+  - Switch
+ha_release: 0.103
+---
+
+The `aten_pe` integration lets you control [ATEN Rack PDUs](https://www.aten.com/eu/en/products/energy-intelligence-pduupsracks/rack-pdu/) from Home Assistant.
+
+In order to use it, SNMP must be enabled on your PDU. For an outlet to appear in Home Assistant, its name must have been configured.
+
+It is recommended to use SNMPv3 to protect your credentials from eavesdropping.
+
+Tested devices:
+  * [PE8324G](https://www.aten.com/eu/en/products/energy-intelligence-pduupsracks/rack-pdu/pe8324/)
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+switch:
+  - platform: aten_pe
+    host: mypdu.lan.
+    authkey: !secret mypdu_authkey
+    privkey: !secret mypdu_privkey
+```
+
+{% configuration %}
+host:
+  description: The IP/host which to control.
+  required: false
+  type: string
+  default: 'localhost'
+port:
+  description: The port on which to communicate.
+  required: false
+  type: string
+  default: '161'
+community:
+  description: community string to use for authentication (SNMP v1 and v2c).
+  required: false
+  type: string
+  default: 'private'
+username:
+  description: Username to use for authentication.
+  required: false
+  type: string
+  default: 'administrator'
+auth_key:
```

Comment:
Key name seems incorrect.

Reply:
Right. I copied and pasted the options from the SNMP switch documentation, but the code deviates from it. I think I should adjust the code and the example above instead.